docs/src/main/asciidoc/elasticsearch.adoc

```diff
@@ -423,138 +437,25 @@ public class FruitService {
     }
 }
 ----
-
-In this example you can note the following:
-
-1. We inject an `ElasticsearchClient` inside the service.
-2. We create an Elasticsearch index request using a builder.
-3. We directly pass the object to the request as the Java API client has a serialization layer.
-4. We send the request to Elasticsearch.
-
```

Comment:
These lines are useful as they explain the code.
If you don't want the anymore, please remove the comments that reference these notes